packages/core/src/shared/sam/build.ts

```diff
@@ -128,7 +129,7 @@ export class BuildWizard extends Wizard<BuildParams> {
         this.arg = arg
         if (this.arg === undefined) {
             // "Build" command was invoked on the command palette.
-            this.form.template.bindPrompter(() => createTemplatePrompter(this.registry))
+            this.form.template.bindPrompter(() => createTemplatePrompter(this.registry, syncMementoRootKey))
```

Comment:
Why build is using `syncMementoRootKey`

Reply:
I am not sure why but this has been like this.